ui.frontend/src/HTTPAPILayer.js

```diff
@@ -84,4 +84,8 @@ export default class HTTPAPILayer {
             xhr.send();
```

Comment:
Please write a test case for this use-case in cypress

Reply:
Added cypress tests where this will be covered: https://github.com/adobe/aem-core-forms-components/pull/642/files#diff-17ac160f6c54427618fc6f8531171af811352cf8dad76516713decfef65b7b29R19